scripts/eval/eval.py

```diff
@@ -68,19 +69,21 @@ def load_peft_model(model_cfg: DictConfig, tokenizer: PreTrainedTokenizerBase,
                     f'Got exception {str(e)} while loading model {model_cfg.name}. {num_retries-retries} retries remaining'
                 )
 
+    assert composer_model_wrapper is not None
```

Comment:
I think this one is just for pyright, so assert is appropriate. That is, the code will never reach this point while `composer_model_wrapper is None`, because it will raise an exception first.